<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="reader_id">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>520</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="borrow_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>1021</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>520</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reader ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>570</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Book ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_id">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>570</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="return_table">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ticket_table">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>321</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Borrow:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Return:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ticket:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="normal_search">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>520</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/chaxun.png</normaloff>icons/chaxun.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>610</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Group search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group_book">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>650</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/book.png</normaloff>icons/book.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="group_reader">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>650</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/readers.png</normaloff>icons/readers.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>650</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>By Readers:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>650</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>By Books:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="MessageBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>normal_search</sender>
   <signal>clicked()</signal>
   <receiver>Search</receiver>
   <slot>normal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>group_reader</sender>
   <signal>clicked()</signal>
   <receiver>Search</receiver>
   <slot>user_group()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>group_book</sender>
   <signal>clicked()</signal>
   <receiver>Search</receiver>
   <slot>book_group()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1007</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>927</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>normal()</slot>
  <slot>user_group()</slot>
  <slot>book_group()</slot>
 </slots>
</ui>
